opwen_email_server/swagger/email-receive.yaml

```diff
@@ -15,27 +32,28 @@ paths:
       summary: Webhook listening to emails received via Sendgrid.
       consumes:
         - multipart/form-data
-      parameters:
-        - $ref: '#/parameters/ClientId'
```

Comment:
ClientId is only used for the `/api/email/sendgrid/{client_id}` endpoint, not for the `/api/email/sendgrid/service` endpoint. The parameter is marked as required in our spec so right now the requests from Sendgrid would fail validation.

Note: if this means it's not possible to use the YAML anchors we'll have to live with the code duplication, but I wonder if it's possible to do something like this:

```yaml
parameters:
  - $ref: '#/parameters/ClientId'
  *SENDGRID_PARAMS
```

Reply:
I messed around with this for a bit and it does not work. Unfortunately I think we'll have to go back to code duplication.

Reply:
Thanks for looking into it.